src/frontend/komponenter/Fagsak/Saksoversikt/Behandlinger.tsx

```diff
@@ -68,21 +68,21 @@ const Behandlinger: React.FC<IBehandlingshistorikkProps> = ({ minimalFagsak }) =
                 )}
             </StyledHeading>
             {behandlinger.length > 0 ? (
-                <table
+                <Table
                     className={classNames('tabell', 'saksoversikt__behandlingshistorikk__tabell')}
```

Comment:
Tror ikke vi trenger klassen `tabell` lenger, den ble vel brukt til å style de gamle tabellene

Reply:
Potensielt ikke den andre klassen heller, sjekk gjerne :)

Reply:
Uten tabell blir den sånn her 🙃
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/87492615-14b2-4e46-9ed1-00817c1e44b4)

Vs sånn her med:
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/fa94bc3c-3093-4dac-abff-958958aac6de)

Tror det kommer fra et nav-frontend-* bibliotek, så vi burde komme oss bort fra det. Jeg kan evt prøve å style det til selv?

Reply:
Alternativt, kanskje det er greit at den blir litt større? Så slipper vi custom styling på komponent fra Aksel?

Reply:
Kan det være fordi den får default størrelse "medium" fra Aksel? Har du prøvd med `size="small"`? Vi bruker det flere andre steder i ba-sak

Reply:
Da blir hele tabellen mindre..
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/bae5fd35-8272-4539-bd2e-dd4635de76ac)

Reply:
Men kanskje det er fint å ha den small, så kan jeg legge på margin top bottom pr rad evt?

Reply:
Aksel kom akkurat med en v5 som fikser dette størrelse-problemet. Får ikke oppgradert til v5 nå fordi det krever endringer på Modal. Har bestemt sammen med Kristine at vi lar klassenavnet ligge sånn det er for nå, så lager vi en egen oppgave på å bytte til Tabell v5, hvor vi kan fjerne classnames

https://aksel.nav.no/produktbloggen/aksel-v5